### PR TITLE
halide: init at 2015.10.22

### PR DESCRIPTION
**Pull request by spwhitt**

### CODE REVIEW

File: pkgs/development/compilers/halide/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    cp tools/GenGen.cpp $out/share/halide/tools
+    cp tools/halide_image.h $out/share/halide/tools
+    cp tools/halide_image_io.h $out/share/halide/tools
+  '' + lib.optionalString stdenv.isDarwin ''
+    install_name_tool -id $out/lib/libHalide.so $out/lib/libHalide.so
+  '';
+
+  meta = {
+    description = "A language for image processing and computational photography";
+    homepage = "http://halide-lang.org";
```
Comment (pSub) [inline pkgs/development/compilers/halide/default.nix]: Nitpick: You don't need quotes around URLs.

File: pkgs/development/compilers/halide/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, llvm }:
+
+stdenv.mkDerivation {
+  name = "halide-2015.10.22";
+
+  src = fetchFromGitHub {
+    owner = "halide";
+    repo = "Halide";
+    rev = "release_2015_10_22";
```
Comment (pSub) [inline pkgs/development/compilers/halide/default.nix]: You could bind the version to a name and use it here by replacing . with _. This is not mandatory but would reduce the number of lines that need to change on an update.

**Comment by pSub** [comment]
Looks good otherwise.

**Comment by pSub** [comment]
The build fails with `Can't find clang or version of clang too old (we need 3.5 or greater)`. See https://s3.amazonaws.com/archive.travis-ci.org/jobs/94800231/log.txt for the full log.

**Comment by spwhitt** [comment]
Thank. That'll be because I only tested on Darwin, and apparently it requires clang. Pushed an update, will ping you when I got it working.

**Comment by pSub** [comment]
Thanks!

**Comment by garbas** [comment]
@spwhitt this doesn't build for me on ``x86_64-linux``

**Comment by garbas** [comment]
@spwhitt ping

**Comment by joachifm** [comment]
Looks abandoned to me.

**Pull request closed without merging**